src/components/VmsList/Vm.js

```diff
@@ -27,7 +27,7 @@ const Vm = ({ vm, icons, os, onStart }) => {
   const icon = getVmIcon(icons, os, vm)
 
   return (
-    <div className={`col-xs-12 col-sm-6 col-md-4 col-lg-3`}>
+    <div className={`col-xs-12 col-sm-6 col-md-4 col-lg-3`} id={`${idPrefix}-box`}>
```

Comment:
~~What if 2 VM's have the same name?~~ Nope, VMs cannot have the same name.  Names are forced unqiue across the entire engine.